cs/ccxt/base/Exchange.Types.cs

```diff
@@ -1912,4 +1912,25 @@ public Option this[string key]
             chains[key] = value;
         }
     }
+}
+
+
+public struct LongShortRatio
+{
+    public Dictionary<string, object>? info;
+    public string? symbol;
+    public Int64? timestamp;
+    public string? datetime;
+    public string? period;
```

Comment:
@Dan-krm why not using the unified term period?

Reply:
I changed period to timeframe